docs/source/getting_started/1_pycharm.rst

```diff
@@ -27,71 +26,116 @@ Project Preparation
 
 #. Start the IDE and create a new project in the ``my-module`` directory.
 
-#. When opening the project, skip the step related to the environment creation: click **Cancel** on **Creating Virtual Environment**:
+#. When creating a project, select the system Python interpreter
 
-.. image:: ../_static/img/dev-env/01-cancel-creating-virtual-environment.png
+.. image:: ../_static/img/dev-env/01-creating-project.png
 
 Setting Up The Interpreter
 --------------------------
+#. Open the project settings with **File > Settings ...** or **Ctrl+Alt+S**. Set **Use compose V2** in **Build, Execution,Deployment > Docker > Tools**
+
+    .. image:: ../_static/img/dev-env/21-select-docker-compose-v2.png
 
 #. Open the project settings with **File > Settings ...** or **Ctrl+Alt+S**.
 
-#. Choose **Project: my-module > Python Interpreter** section.
+#. Select **Project: my-module > Python Interpreter** section.
 
-#. Add a new **Python Interpreter** by clicking the **Add Interpreter**, and choosing **On Docker...**:
+#. Add a new **Python Interpreter** by clicking the **Add Interpreter**, and selecting **On Docker Compose...**:
 
     .. image:: ../_static/img/dev-env/02-setup-python-interpreter.png
 
 #. In the modal window:
 
-  * **[Step 1 of 3]**: Choose Dockerfile: ``docker/Dockerfile.x86`` or ``docker/Dockerfile.l4t``, if you are on Jetson, and set up Context folder (should be set to the root of the project). In the **Optional** section you have to specify an **Image tag**:
+  * **[step 1 of 4]**: Click on the three dots under **Server** and set up access to the docker service
 
-    .. warning::
-        If you do not specify **Image tag**, docker images and container will not update correctly.
+    .. image:: ../_static/img/dev-env/20-setup-docker.png
 
-    .. image:: ../_static/img/dev-env/03-setup-docker-build.png
+  * **[Step 2 of 4]**: Select Configuration file: ``docker-compose.x86.yml`` (``docker-compose.l4t.yml`` for Jetson) and select the service **module**:
 
-  * **[Step 2 of 3]**: Make sure docker build is successful.
+    .. image:: ../_static/img/dev-env/03-setup-python-interpreter-docker-compose-module.png
 
-  * **[Step 3 of 3]**: Keep the default python interpreter (python3) and click **Create**:
+  * **[Step 3 of 4]**: Click **Next** button, make sure docker build is successful and then again click **Next** button.
+
+  * **[Step 4 of 4]**: Keep the default python interpreter (python3) and click **Create**:
 
     .. image:: ../_static/img/dev-env/04-setup-interpreter-in-docker.png
 
-4. Make sure that the ``savant`` package is in the package list, click **OK**:
+#. Make sure that the ``savant`` package is in the package list, click **OK**:
```

Comment:
Try to build the docs, this point has number 1. Please check the order of the numbers.

Reply:
Fixed list indentation, ordered list numbers are correct now.